fr/strings.po

```diff
@@ -855,47 +855,47 @@ msgstr "Mot de Passe du Cluster"
 
 #. i18n: keyword: ItemsInventoryTitle
 msgid "Your Items Inventory"
-msgstr ""
+msgstr "Votre inventaire"
 
 #. i18n: keyword: ItemsDialogRarity
 msgid "Rarity:"
-msgstr ""
+msgstr "Rarité :"
 
 #. i18n: keyword: ItemsDialogRedeemed
 msgid "Redeemed:"
-msgstr ""
+msgstr "Échangé :"
 
 #. i18n: keyword: ItemsFilterRarity
 msgid "Rarity"
-msgstr ""
+msgstr "Rarité"
 
 #. i18n: keyword: RollbackDialogConfirmation
 msgid "Are you sure you want to rollback this transaction?"
-msgstr ""
+msgstr "Voulez-vous vraiment annuler cette transaction ?"
 
 #. i18n: keyword: RollbackDialogNotEnoughCurrency
 msgid "You don't have enough currency for this rollback! You need another"
-msgstr ""
+msgstr "Vous n'avez pas assez de points pour ce retour en arrière ! Vous avez besoin d'un autre"
```

Comment:
I think the "You need another" is for "You need another 10 BOLTS"

In that case, it would be "Vous avez besoin de ..."